website/test/e2e/api_spec.js

```diff
@@ -133,4 +133,11 @@ describe('Api', function() {
     expect(browser.getCurrentUrl()).
         toMatch(/api\?view=webdriver.WebDriver.prototype.executeAsyncScript/);
   });
+
+  it('should sort the menu to put webdriver docs next to the relavant ' +
```

Comment:
If the files were arranged differently the location of `browser` and `by` in the docs might be switched, but that shouldn't break the tests.  This test is designed to make sure that `webdriver.WebDriver` comes directly after `browser`, since that's how the inheritance works, but not care about the absolute order of everything.